src/cascadia/TerminalCore/TerminalApi.cpp

```diff
@@ -169,3 +169,43 @@ bool Terminal::SetColorTableEntry(const size_t tableIndex, const DWORD dwColor)
     _buffer->GetRenderTarget().TriggerRedrawAll();
     return true;
 }
+
+bool Terminal::SetCursorStyle(const DispatchTypes::CursorStyle cursorStyle)

```

Comment:
nit:This method needs a method description on it

Reply:
Done!